### ISSUE TITLE
Hipercard type support

### ISSUE THREAD

**Issue by sibelius** [braintree/credit-card-type#58]
Based on this one https://github.com/moip/credit-card-validator/blob/4786855a/src/main/java/br/com/moip/creditcard/HipercardCreditCard.java#L6-L7

**Comment by crookedneighbor** [comment]
Closed as a duplicate of https://github.com/braintree/credit-card-type/issues/57

We must have official documentation before we can add additional card types to this library. I'm happy to re-open this issue if you can provide the documentation.

**Comment by crookedneighbor** [comment]
v6.2.0 allows you to add custom card brands. See https://github.com/braintree/credit-card-type#adding-card-types for documentation.

**Comment by sibelius** [comment]
Great to know, tks
